benchmarks/src/main/scala/caliban/GraphQLBenchmarks.scala

```diff
@@ -361,8 +369,7 @@ class GraphQLBenchmarks {
           new NoUndefinedVariables,
           new NoUnusedFragments,
           new NoUnusedVariables,
-          // new OverlappingFieldsCanBeMerged,
-          new experimental.OverlappingFieldsCanBeMerged,
+          new OverlappingFieldsCanBeMerged,
```

Comment:
Is the experimental one the new default?

Reply:
I believe so, `experimental.OverlappingFieldsCanBeMerged` doesn't exist anymore (was probably removed in Sangria 3.x)

Reply:
Thanks!